lib/main.js

```diff
@@ -113,9 +113,30 @@ exports.mjpage = function(htmlstring, configOptions, typesetOptions, callback) {
         window.ascii.config.doc = document;
         window.ascii.PreProcess();
     }
+    //normalize type strings
+    const scripts = document.querySelectorAll('script[type^="math/"]');
+    for (let i = 0; i < scripts.length; i++) {
+        const script = scripts[i];
+        let format = script.getAttribute("type").toLowerCase().slice(5);
+        if (format === "inline-tex") {
+            format = "math/inline-TeX";
+            // via https://github.com/mathjax/MathJax/blob/master/unpacked/jax/input/TeX/jax.js#L2168
+        } else if (format.match(/tex((;|\s*|\n*)mode\s*=\s*display)?/i)) {
+            format = "math/TeX";
+        } else if (format === "mathml") {
+            format = "math/MathML";
+        } else if (format === "mathml-block") {
+            format = "math/MathML-block";
+        } else if (format === "asciimath") {
+            format = "math/AsciiMath";
+        } else {
+            console.log("Unknown format " + format);
```

Comment:
This is a good point but you should really remove that node (it will still get processed later). But removing nodes from nodelists requires conversion to an array. Makes me wonder...